src/controllers/addflowerController.js

```diff
@@ -0,0 +1,19 @@
+const addFlower = require('../database/queries/addflower')
+exports.addone = (request,response)=>{
+
+    
+const input=request.body
+
+
+    addFlower(input.name,input. arabicname,input.imgUrl,input.description,(err,res)=>{
```

Comment:
use destructuring `const {name,arabicname,imgUrl,description} =req.body; ` then pass the varaibles to `addFlower` function

Reply:
resolve comment